app/admin/partners/page.tsx

```
@@ -0,0 +1,1283 @@
+    if (activeTab !== "all" && partner.status !== activeTab) {
+      return false;
+    }
+    
+    // Apply search filter
+    if (searchQuery) {
+      const query = searchQuery.toLowerCase();
+      return (
+        partner.name.toLowerCase().includes(query) ||
+        partner.industry.toLowerCase().includes(query) ||
+        partner.location.toLowerCase().includes(query) ||
+        (partner.description && partner.description.toLowerCase().includes(query))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)